README.md

```diff
@@ -2,11 +2,77 @@
   <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png" />
 </p>
 
-<p align="center">
-  <strong>Our Github Action publishes your documentation to your project's Github Wiki!</strong>
-</p>
+<h3 align="center">Publish Github Wiki Action</h3>
+<p align="center">This action helps syncs your project's documentation with your project's Github Wiki.</p>
+
+-------
+
+## Usage
+
+Add the following workflow as a step in your existing workflow or a new one, replacing `USER_NAME` and `USER_EMAIL` with your account/bot information.
+
+```yml
+- name: Publish Github Wiki
+  uses: nimblehq/publish-github-wiki-action@v1.0
+  env:
```

Comment:
Should we turn these environment variables into inputs?
In other actions, they often pass data to the action via inputs, and we can mark that input required, or set the default value. 
What do you think?
You can check [the example here](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action)

Reply:
Thanks for the suggestion! I've updated the workflow to pass data via inputs 0c418d9c764702f517d662624c7414d6f0ceaef1
And updated the usage instructions 😄